File: PYTHON/Program/Monitor.py
from OutputDevice import OutputDevice

class Monitor(OutputDevice):
    def __init__(self, refreshRate=0, resolution="", jenisOutput="", outputLatency="", merk="", nama=""):
        super().__init__(jenisOutput, outputLatency, merk, nama)
        self.refreshRate = refreshRate
        self.resolution = resolution

    def setRefreshRate(self, refreshRate):
        self.refreshRate = refreshRate

    def setResolution(self, resolution):
        self.resolution = resolution

    def getRefreshRate(self):
        return self.refreshRate

    def getResolution(self):
        return self.resolution

File: JAVA/Program/Monitor.java
package Program;
public class Monitor extends OutputDevice {
    private int refreshRate;
    private String resolution;

    public Monitor() {
    }

    public Monitor(int refreshRate, String resolution, String jenisOutput, String outputLatency, String merk, String nama) {
        super(jenisOutput, outputLatency, merk, nama);
        this.refreshRate = refreshRate;
        this.resolution = resolution;
    }

    public void setRefreshRate(int refreshRate) {
        this.refreshRate = refreshRate;
    }

    public void setResolution(String resolution) {
        this.resolution = resolution;
    }

    public int getRefreshRate() {
        return this.refreshRate;
    }

    public String getResolution() {
        return this.resolution;
    }
}
